cpp/StrangeCharacters/test/CharactersTest.cpp

```diff
@@ -50,6 +33,11 @@ std::vector<std::unique_ptr<Character>> allTestData()
     will->Nemesis = mindflayer.get();
     dustin->Nemesis = demadog.get();
```

Comment:
very small thing I think you missed - eleven also has a nemesis

Reply:
That's just a trick that Github plays - it's in the file, but the diff doesn't show it. Click on the arrows on the left to see it.